.github/workflows/add-to-dev-prod-project.yml

```
@@ -0,0 +1,42 @@
+name: add-to-dev-prod-project
+
+on:
+  issues:
+    types: [opened, labeled]
+  pull_request:
+    types: [opened, labeled]
+
+jobs:
+  add-to-dev-prod-project:
+    if: contains(github.event.issue.labels.*.name, '@dev-productivity')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get project data
+        run: |
+          gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
+                  id
+                  fields(first:20) {
+                    nodes {
+                      id
+                      name
+                      settings
+                    }
+                  }
+                }
+              }
+            }' -f org=gradle -F number=17 > project_data.json
```

```suggestion
            }' -F org=gradle -F number=17 > project_data.json
```

Shouldn't this be `F` in both cases?